ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -56,7 +56,14 @@ export default function CustomSpendingCap({
     } else if (inputNumber > currentTokenBalance) {
       return {
         className: 'custom-spending-cap__higherValue',
-        description: t('inputLogicHigherNumber'),
+        description: t('inputLogicHigherNumber', [
+          <b key="custom-spending-cap-input-bold">
```

Comment:
We'll want to use the `<Typography>` component here, much like on line 46.  We're trying to avoid formatting tags like `<b>`

Reply:
Replaced